rust/lance-index/src/scalar/inverted.rs

```diff
@@ -181,25 +179,32 @@ impl ScalarIndex for InvertedIndex {
         let mut token_set = self.tokens.clone();
         let mut invert_list = self.invert_list.clone();
         let mut docs = self.docs.clone();
+        let stopword_filter =
+            tantivy::tokenizer::StopWordFilter::new(tantivy::tokenizer::Language::English).unwrap();
         let mut tokenizer = tantivy::tokenizer::TextAnalyzer::builder(
-            tantivy::tokenizer::SimpleTokenizer::default(),
+            stopword_filter.transform(tantivy::tokenizer::SimpleTokenizer::default()),
         )
         .build();
```

Comment:
I assume we'll make this all configurable later, right?

Reply:
Yes, just make it work now